Misc/NEWS.d/next/Library/2023-04-26-18-12-13.gh-issue-103636.-KvCgO.rst

```diff
@@ -0,0 +1 @@
+module-level attribute January and February is deprecated
```

Comment:
News notes can use markup, and should mention the module they are talking about 🙂